readme.md

```diff
@@ -240,6 +240,8 @@ This section details projects that have integrated Lighthouse. If you're working
 
 * **[Calibre](https://calibreapp.com)** - Calibre is a web performance monitoring tool running Lighthouse continuously or on-demand via an API. Test using emulated devices and connection speeds from a number of geographical locations. Set budgets and improve performance with actionable guidelines. Note that Calibre is a paid product with a free 14-day trial.
 
+* **[Greta Lighthouse](https://lighthouse.greta.io/)** - Greta Lighthouse is a tool that lets you run free Google Lighthouse tests from anywhere in the world. It generates a report providing insights into your users perceived performance and recommendations on how to improve. The tool considers parameters such as location and type of device to simulate real user conditions. [Greta](https://greta.io/) also provides a new type of platform that helps companies understand, control and improve their users’ experience via an innovative approach to content delivery.
```

Comment:
can probably remove the "generates a report providing" since all the Lighthouse report integrations should be similar on that front :)

mind adding a quick paid plan caveat for the platform plug if there's 💰 involved too?

Reply:
The Greta platform is still in closed beta, So price plane is not really relevant at this time. Can update as that progresses?

Reply:
That SGTM to me.

Reply:
oh cool yeah if there's no 💵  involved yet, no need just assumed :)